### PR TITLE
Add Wikibase Repo entityNamespaces to ManageWiki

### PR DESCRIPTION
**Pull request by Universal-Omega**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -4335,6 +4335,12 @@
 	'wmgWikibasePropertyNamespaceID' => [
 		'default' => 120
 	],
+	'wmgWikibaseRepoItemNamespaceID' => [
```
Comment (ugochimobi) [inline LocalSettings.php:4338]: Please It should be "0" now

Comment (Universal-Omega) [reply LocalSettings.php:4338]: The default is being maintained. It will be changeable from ManageWiki once this is merged and live.

Comment (ugochimobi) [reply LocalSettings.php:4338]: Okay then
On manage wiki It's actually 0 but it's that $wgWBRepoSettings['entityNamespaces']['item'] = NS_MAIN; that needs to be configured

**Comment by ugochimobi** [comment]
It's now working perfectly brother! Many thanks

**Comment by ugochimobi** [comment]
@Universal-Omega  In FAMEPedia, on the sidebar of articles with FAMEData items, It's supposed to be showing "FAMEData Item" instead of "Wikibase Item"
How do we configure that?

**Pull request merged**